app/connectors/SubmitVatReturnConnector.scala

```diff
@@ -41,10 +38,10 @@ class SubmitVatReturnConnector @Inject()(val http: HttpClient, val appConfig: Mi
         "Environment" -> appConfig.desEnvironment,
```

Comment:
You need to remove these headers from the `HeaderCarrier` and create a separate val to put them in (you also need to include the Authorization header in this new list). This new val is going to be passed into the `POST` function below as the headers parameter.